sbtc-core/src/signer/mod.rs

```diff
@@ -143,31 +125,3 @@ impl<S: Sign + Coordinate> Keys for Signer<S> {
         todo!()
     }
 }
-
```

Comment:
Did you mean to remove the skeleton implementations?

Reply:
Yes I did. The coordinator should actually be implemented on the frost-coordinator, fire-coordinator, roast-coordinator level and then passed to the Signer struct.